modules/opensimplex/noise.h

```diff
@@ -0,0 +1,62 @@
+#ifndef OPENSIMPLEX_NOISE_H
```

Comment:
Copyright header needed here too. The guard should likely be renamed to match the class and filename (so `SIMPLEX_NOISE_H` once renamed to `simplex_noise.h`).